tests/check/check_h26xsigned_sign.c

```diff
@@ -636,6 +639,8 @@ START_TEST(two_completed_seis_pending)
   // Check if there are no more SEIs remaining in the buffer
   ck_assert_int_eq(sei_size_3, 0);
   // Verify the transfer order of NALUs
+  // Expect |sei_size_1| to be less than |sei_size_2|
+  // as the first SEI is at least 32 bytes smaller than the second.
```

Comment:
This is not the correct reason. The reason is that the second SEI includes more one more hash.